node/src/GlideClient.ts

```diff
@@ -330,4 +331,33 @@ export class GlideClient extends BaseClient {
     public lolwut(options?: LolwutOptions): Promise<string> {
         return this.createWritePromise(createLolwut(options));
     }
+
+    /**
+     * Loads a library to Valkey.
+     *
+     * See https://valkey.io/commands/function-load/ for details.
+     *
+     * since Valkey version 7.0.0.
```

Comment:
The existing docs put the since section after the `@returns` or `Command Response` section, can we either align with that or, if we think its better here, align on putting it here going forward and refactor the existing docs?

Reply:
We already have API where `since` section located in different places, so I don't care much about it. We have a task to introduce `@since` tag and groom it thoughout all the docs.